v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,131 @@
+
+With parallel statement execution, however, multiple SQL statements within a transaction are executed at the same time, thereby reducing the aggregate latency over multiple SQL statements. 
+
+## When to use Parallel Statement Execution
+
+SQL statements within a single transaction can be executed in parallel if the statements are [independent](parallel-statement-execution.html#independent-sql-statements). CockroachDB considers SQL statements within a single transaction to be independent if their execution can be safely reordered without affecting their results. 
+
+## How Parallel Statement Execution Works
+
+Let's understand how sequential and parallel execution works in the following scenario:
+
+
+- Suppose we want to update a user's last name, favorite movie, and favorite song on a social networking application.
+- The database has three tables that need to be updated: `users`, `favorite_movies`, and `favorite_songs`.
+
+Then the traditional transaction to update the user's information is as follows:
+
+~~~ sql
+> BEGIN;
+> UPDATE users SET last_name = 'Smith' WHERE id = 1;
+> UPDATE favorite_movies SET movies = 'The Matrix' WHERE user_id = 1;
+> UPDATE favorite_songs SET songs = 'All this time' WHERE user_id = 1;
+> COMMIT;
+~~~
+
+While executing the SQL statements in the transaction sequentially, the server sends a return value after executing a statement. The client can send the next statement to be executed only after it receives the return value of the previous statement. This is often described as a "conversational API," as demonstrated by the following conceptual diagram:
+
+<img src="{{ 'images/Sequential_Statement_Execution.png' | relative_url }}" alt="CockroachDB Parallel Statement Execution" style="border:1px solid #eee;max-width:100%" />
+
+The SQL statements in our sample scenario can be executed in parallel since they are independent of each other. To execute statements in parallel, the client should be able to send the next statement to be executed without waiting for the return value of the earlier statement. In CockroachDB, on appending the `RETURNING NOTHING` clause with SQL statements,  the server sends an acknowledgment immediately, instead of waiting to complete the statement execution and sending the return value to the client. The client sends the next statement to be executed on receiving the acknowledgment. This allows CockroachDB to execute the statements in parallel. The statements are executed in parallel until CockroachDB encounters a **barrier statement**. A barrier statement is any statement without the `RETURNING NOTHING clause. The server executes a barrier statement sequentially.
```

Comment:
Need a closing backtick after the last `RETURNING NOTHING`.

Reply:
Fixed.